source/backend/api/Services/DocumentService.cs

```diff
@@ -152,6 +152,7 @@ public async Task<DocumentUploadResponse> UploadDocumentAsync(DocumentUploadRequ
                 {
                     var retryPolicy = Policy<ExternalResponse<DocumentDetailModel>>
                         .HandleResult(result => result.HttpStatusCode != HttpStatusCode.OK || result.Payload.FileLatest == null)
+                        .Or<System.Text.Json.JsonException>()
```

Comment:
based on fields like size being returned as null in some situations - may not be necessary with model updates.